recipes/fiscalyear/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
You license is already being packaged in Pypi, you don't need this file, please delete it

Reply:
I do not believe the license is included in the tarball on Pypi.  After removing it from the recipe, the build failed with the following error:

```
ValueError: License file given in about/license_file (C:\projects\staged-recipes\recipes\fiscalyear\LICENSE) does not exist in source root dir or in recipe root dir (with meta.yaml)
```

Reply:
That is true, sorry. I might downloaded the wrong project when I was checking it. My apologies about it.
Could you please open a pull request or create an issue on this github's project to also package the LICENSE?
Please revert your last commit then, sorry

Reply:
PR to add license to the tarball: https://github.com/adamjstewart/fiscalyear/pull/7